public/app/features/alerting/unified/components/GrafanaAlertmanagerDeliveryWarning.tsx

```diff
@@ -0,0 +1,46 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src';
+import { Alert, useStyles2 } from '@grafana/ui/src';
+
+import { AlertmanagerChoice } from '../../../../plugins/datasource/alertmanager/types';
+import { GRAFANA_RULES_SOURCE_NAME } from '../utils/datasource';
+
+interface GrafanaAlertmanagerDeliveryWarningProps {
+  alertmanagerChoice?: AlertmanagerChoice;
+  currentAlertmanager: string;
+}
+
+export function GrafanaAlertmanagerDeliveryWarning({
+  alertmanagerChoice,
+  currentAlertmanager,
+}: GrafanaAlertmanagerDeliveryWarningProps) {
+  const styles = useStyles2(getStyles);
+
+  if (currentAlertmanager !== GRAFANA_RULES_SOURCE_NAME) {
+    return null;
+  }
+
+  if (alertmanagerChoice !== AlertmanagerChoice.External) {
+    return null;
+  }
+
+  return (
+    <Alert title="Grafana alerts are not delivered to built-in Grafana Alertmanager">
+      Grafana is configured to send alerts to external Alertmanagers only. Changing Grafana Alertmanager configuration
+      will not affect your alerts!
+      <div className={styles.adminHint}>
+        This configuration can be changed in the Alerting Admin section. If you do not have access to the Admin section
```

Comment:
We are using configuration twice in this warning. Maybe we could differentiate them saying something like:
`Alert manager selected to send alerts can be changed in the Alerting Admin section.`

Reply:
As above. Grafana alerts are not delivered to the Grafana Alertmanager. Grafana is configured to send alerts to external alertmanagers only.